net/rhp/rpc_test.go

```diff
@@ -0,0 +1,88 @@
+package rhp
+
+import (
+	"bytes"
+	"math/rand"
+	"reflect"
+	"testing"
+	"testing/quick"
+
+	"go.sia.tech/core/types"
+)
+
+func (p *HostSettings) Generate(rand *rand.Rand, size int) reflect.Value {
```

Comment:
`// Generate implements quick.Generator.`

also, you can use `func (*HostSettings)` when the receiver variable is unused.